### PR TITLE
Linked Github profile with Username

### LINKED ISSUE #7: [Web- html,css,javascript] Link Username to github profile
**Title**
Link Username to github profile

**Project link**
[Click here](https://github.com/DevilsAutumn/Hacktoberfest22/tree/main/Web/Html%2CCSS%2CJavascript/github-profiles)

**Domain**
Add 'x' inside [ ] to cross check box.

- [ ] App with Flutter
- [ ] App with Java
- [ ] App with Kotlin
- [x] Web with Html,css javascript
- [ ] Web with React

**Please describe your feature.**
After searching for profile ,clicking on the username, users should be redirected to the profile on github.

**Describe the solution you'd like**
NA

**Additional context**
NA

### PR DESCRIPTION
**Pull request by kunal00000**
# Description

linked GitHub username with profile.

Fixes #7 

Replace `issue_no` with the issue number which is fixed in this PR

## Type of change

_Please delete options that are not relevant._

- [ ] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have created a helpful and easy to understand `README.md`
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

File: Web/Html,CSS,Javascript/github-profiles/script.js [hunk 1]
```diff
@@ -28,6 +28,7 @@ async function getRepos(username) {
 }
 
 function createUserCard(user) {
+    const userURL = user.html_url
```
Comment (DevilsAutumn) [inline Web/Html,CSS,Javascript/github-profiles/script.js:31]: Github user does not have any attribute named "html_url". Its only url and that too won't work here.

File: Web/Html,CSS,Javascript/github-profiles/script.js [hunk 2]
```diff
@@ -36,7 +37,7 @@ function createUserCard(user) {
       <img src="${user.avatar_url}" alt="${user.name}" class="avatar">
     </div>
     <div class="user-info">
-      <h2>${userID}</h2>
+      <a style="color:rgb(238,238,238); text-decoration:none;" href="${userURL}"><h2>${userID}</h2></a>
```
Comment (DevilsAutumn) [inline Web/Html,CSS,Javascript/github-profiles/script.js:40]: You just have to redirect to the link: www.github.com/<username> where username can be extracted from user object.

Comment (kunal00000) [reply Web/Html,CSS,Javascript/github-profiles/script.js:40]: Screenshot 2022-10-04 at 11 00 58 AM


Yes, first I thought would have somewhat do something like adding username in "www.github.com/" by extracting it from user object but when I checked API it was providing direct link for that and found that with "html_url" in the user object.
It's working properly I have tested it.

**Comment by DevilsAutumn** [comment]
Cool 👍

**Pull request merged**